scss/detail.scss

```diff
@@ -0,0 +1,85 @@
+.book-title {
+  height: 39px;
```

Comment:
sort the properties alphabetically. Also try to use a base for properties height, with, margin and padding, I suggest 5px so try that all measures are a multiple of this, so here you can put 40px;